### DISCUSSION TITLE
Hide "Current progress"

### DISCUSSION THREAD

**Discussion by gotarik** [ppy/osu#29124] [Ideas] [answered]
Why can we hide hud, but can't hide this thing that shows accuracy and grade
![osu_2024-07-27_01-13-23](https://github.com/user-attachments/assets/7f825c0b-363c-45ed-9bf6-63dd91881194)

**Comment by peppy** [comment, marked as answer]
It's during a break. Hiding would be redundant because it's not distracting from your gameplay.
